src/pudl_archiver/metadata/sources.py

```diff
@@ -21,8 +21,81 @@
 ADDL_CONTRIBUTORS: dict[str, dict[str, str]] = {}
 
 NON_PUDL_SOURCES: dict[str, Any] = {
+    "doelead": {
+        "title": "DOE LEAD -- Low Income Energy Affordability Data",
+        "path": "https://www.energy.gov/scep/low-income-energy-affordability-data-lead-tool",
+        "description": (
+            "This archive includes the data behind the Department of Energy's (DOE)"
+            "Low Income Energy Affordability Data (LEAD) tool. The LEAD tool is an"
+            "online, interactive platform that helps users make data-driven decisions"
+            "on energy goals and program planning by improving their understanding of"
+            "low-income and moderate-income household energy characteristics. The LEAD"
+            "Tool offers the ability to select and combine geographic areas (state,"
+            "county, city and census tract) into one customized group so users can see"
+            "the total area for their customized geographies (e.g., specific service"
+            "territories)."
+        ),
+        "working_partitions": {"years": [2022, 2018]},
```

Comment:
Not sure if we depend on this programmatically anywhere, but we've always listed sequential partitions in ascending order.

```suggestion
        "working_partitions": {"years": [2018, 2022]},
```